operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakIngressTest.java

```diff
@@ -188,4 +193,81 @@ public void testMainIngressDurability() {
                     assertThat(k8sclient.network().v1().ingresses().inNamespace(namespace).list().getItems().size()).isEqualTo(0);
                 });
     }
+
+    @Test
+    public void testCustomIngressDeletion() {
+
+        Keycloak defaultKeycloakDeployment = K8sUtils.getDefaultKeycloakDeployment();
+        String kcDeploymentName = defaultKeycloakDeployment.getMetadata().getName();
+        Resource<Ingress> customIngressDeployedManuallySelector = null;
+        Ingress customIngressCreatedManually;
+
+        try {
+
+            customIngressCreatedManually = createCustomIngress(kcDeploymentName, namespace, 8443);
+
+            customIngressDeployedManuallySelector = k8sclient
+                    .network()
+                    .v1()
+                    .ingresses()
+                    .inNamespace(namespace)
+                    .withName(customIngressCreatedManually.getMetadata().getName());
+
+            Awaitility.await().atMost(1, MINUTES).untilAsserted(() -> {
+                assertThat(k8sclient.network().v1().ingresses().inNamespace(namespace).list().getItems().size()).isEqualTo(1);
+            });
+
+            Log.info("Redeploying the Keycloak CR with default Ingress disabled");
+            defaultKeycloakDeployment = K8sUtils.getDefaultKeycloakDeployment();
```

Comment:
Just a non-blocking nitpick. We don't need it as this variable is already assigned here:
https://github.com/keycloak/keycloak/blob/89586ebaf4b6ace35d3b9f376975f67095ee5bc2/operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakIngressTest.java#L200

Reply:
Removed.

Thanks.